### DISCUSSION TITLE
Server Side Resumability (Qwik)

### DISCUSSION THREAD

**Discussion by rchavezj** [sveltejs/kit#8488] [Ideas]
I was wondering if the svelte kit team have plans implementing a SSR Resumability similar to Qwik?

**Comment by pfz** [comment]
From Miško Hevery's perspective as CTO at builder.io / Qwik, here is his answer : https://youtu.be/Zddw6qy5pf0?t=3123.
(TLDW; It's hardly possible, essentially because of the ecosystem.)

As I am not sufficiently skilled, I cannot say if it is debatable.

**Comment by royeradames** [comment]
This can be a killer feature. I will motive my team significantly to move from Angular.ts to Svelte

For anyone who is doing research, this video gives great demos on the pros and comparison: https://www.youtube.com/watch?v=0dC11DMR3fU
